pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/analytics/metrics/MetricsCollector.kt

```diff
@@ -31,9 +30,10 @@ import java.io.IOException
  * - Dropped video frames
  * - Surface size
  */
-class MetricsCollector private constructor(
-    private val timeProvider: () -> Long,
-) {
+class MetricsCollector(
+    private val sessionManager: PlaybackSessionManager,
+    private val timeProvider: () -> Long = { System.currentTimeMillis() },
+) : PillarboxAnalyticsListener {
```

Comment:
Same here. Now `MetricsCollector` will show every method of `PillarboxAnalyticsListener` in the documentation: https://android.pillarbox.ch/api/pillarbox-player/ch.srgssr.pillarbox.player.analytics.metrics/-metrics-collector/index.html
So the actual methods will be harder to find.